pages/index.vue

```diff
@@ -69,6 +69,13 @@ const config = useRuntimeConfig();
         <ShortSectionTile imageSrc="" title="" defaultTile />
       </div>
     </section>
+
+    <section>
+    <div>
+    <Slider :slides="slides" />
```

Comment:
@AgnieszkaIwan is this code needed right now? Besides, shouldn't this component be imported and named as SliderPlusDescription?